sources/pmwebapi.go

```diff
@@ -88,57 +88,44 @@ func (p *pcpPmwebapiSource) Update(ch chan<- prometheus.Metric) error {
 	}
 	return nil
 }
-func newPcpSource() (PcpSource, error) {
-
-	var p pcpPmwebapiSource
 
-	tokenResp, err := http.Get("http://localhost:44323/pmapi/context?hostname=localhost")
+func getRequest(url string) []byte {
+	getResponse, err := http.Get(url)
 	if err != nil {
 		log.Fatal("NewRequest: ", err)
 	}
-	defer tokenResp.Body.Close()
+	defer getResponse.Body.Close()
+	getResponseBody, _ := ioutil.ReadAll(getResponse.Body)
```

Comment:
I realize I haven't pointed this out before, but probably best to check for an error here. I've run into issues in the past where I got an http response with status 200, but the body was invalid and this line failed. I would just mirror what you have above with something like:
```go
getResponseBody, err := ioutil.ReadAll(getResponse.Body)
if err != nil {
    log.Fatal("Unable to read HTTP response:", err)
}
```